portfolio.html

```diff
@@ -206,18 +207,52 @@ <h2>About Me</h2>
         </div>
 
         <ul>
-            <li class="text work">CodeKit</li>
-            <li class="text work">Github</li>
-            <li class="text work">CodePen</li>
-            <li class="text work">Gitlab</lli>
-            <li class="text work">Terminal</li>
-
+            <li class="text work">Database Management</li>
+            <li class="text work">Version Control</li>
+            <li class="text work">CLI</li>
+            <li class="text work">API Design</li>
+            <li class="text work">Web Development</li>
         </ul>
-
     </ul>
-
 </section>
-
-
+<section class="form">
+    <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/xrgvyznd" method="post">
+        <ul>
+            <li>
+                <label for="username">
+                <input id="username" type="text" name="username" placeholder="Full Name" maxlength="30" required>
+            </li>
+            <li>
+                <label for="email"></label>
+                <input id="email"  name="email" type="email"placeholder="Email Address" required>
+            </li>
+            <li>
+                <label for="textarea"></label><textarea name="message" id="textarea" rows="7" cols="100" maxlength="500" placeholder="Write me something..."></textarea>
```

Comment:
- [ ] As the previous code reviewer mentioned and the project required, kindly add the `require` attribute to the `textarea` too. The user cannot submit the form if any of the input fields are empty.
 - 
<img width="520" alt="image" src="https://user-images.githubusercontent.com/61919617/228305173-c6720683-2f46-4fd5-bcb9-2a9227acb18b.png">